vars/fhcapNode.groovy

```diff
@@ -59,7 +59,7 @@ def call(Map parameters = [:], body) {
                 env.PATH = "${PATH}:/home/jenkins/bin"
                 env.FHCAP_CFG_FILE = configFile
 
-                sh "yes | fhcap setup --repos-dir ${WORKSPACE} --fh-src-dir ${WORKSPACE}"
+                sh "yes | fhcap setup --repos-dir ${WORKSPACE} --fh-src-dir ${WORKSPACE} --knife-dir ${WORKSPACE} --empty-config"
```

Comment:
Need to make these options configurable since we cant currently add the "--empty-config" option all the time.

Reply:
The cluster pipelines need modified before we can add this all the time, i intend to add an option pass these in, so we can have it use them when we set them explicitly. Eventually this should be the default though.